### PR TITLE
WINC-708: Collect containerd logs on Windows instances

### PR DESCRIPTION
**Pull request by jrvaldes**
This PR adds the containerd runtime logs in the `must-gather` for Windows nodes.

### CODE REVIEW

File: collection-scripts/gather_windows_node_logs [hunk 1]
```diff
@@ -3,11 +3,12 @@ BASE_COLLECTION_PATH="/must-gather"
 WINDOWS_NODE_LOGS=$BASE_COLLECTION_PATH/host_service_logs/windows
 
 # Services logging to WinEvent log
-SERVICES=(docker)
+SERVICES=(docker containerd)
 
 # Logfile list
 LOGS=(kube-proxy/kube-proxy.exe.INFO kube-proxy/kube-proxy.exe.ERROR kube-proxy/kube-proxy.exe.WARNING)
 LOGS+=(hybrid-overlay/hybrid-overlay.log kubelet/kubelet.log)
+LOGS+=(containerd/containerd.log)
```
Comment (jrvaldes) [inline collection-scripts/gather_windows_node_logs:11]: containerd logs location is defined in [openshift/windows-machine-config-operator#891](https://github.com/openshift/windows-machine-config-operator/pull/891/files#diff-4d8fe3bb66a872f0e4a277f820d60d2efa722f09aa89516be74b98c06b57f3bdR423)

Comment (sferich888) [reply collection-scripts/gather_windows_node_logs:11]: How many logs is this? What is the archive size increase that results from gathering these logs?

Comment (sferich888) [reply collection-scripts/gather_windows_node_logs:11]: Can the collection be limited to a time range? see: https://github.com/openshift/must-gather/blob/master/collection-scripts/gather_service_logs_util#L28 for examples of where / how we do this in other places.

Comment (jrvaldes) [reply collection-scripts/gather_windows_node_logs:11]: > How many logs is this? What is the archive size increase that results from gathering these logs?

Good question, [here](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_windows-machine-config-operator/981/pull-ci-openshift-windows-machine-config-operator-master-vsphere-e2e-operator/1502666441088831488/artifacts/vsphere-e2e-operator/windows-e2e-operator-test-with-custom-vxlan-port/artifacts/nodes/e2e-wm-7kwvd/) you can find logs files for a Windows node.

Comment (jrvaldes) [reply collection-scripts/gather_windows_node_logs:11]: > Can the collection be limited to a time range? see: https://github.com/openshift/must-gather/blob/master/collection-scripts/gather_service_logs_util#L28 for examples of where / how we do this in other places.

Agree, we should apply time range constraints to the collected logs. However, it is out of the scope of this PR. Happy to address it in a separate PR. WDYT?

Comment (davemulford) [reply collection-scripts/gather_windows_node_logs:11]: @jrvaldes the example logs location you linked does not have a containerd.log. I think the question Eric is trying to address is: _how chatty is containerd?_

I'm fine with addressing the time range later, as only clusters with Windows nodes will have this added containerd log, when containerd is available.

Comment (jrvaldes) [reply collection-scripts/gather_windows_node_logs:11]: > how chatty is containerd?
 IDK, yet. containerd is a WIP feature targeted for 4.11 ([WINC-505](https://issues.redhat.com/browse/WINC-505))

Comment (RickJWagner) [reply collection-scripts/gather_windows_node_logs:11]: Reviewed, no comments outside of the issues raised by Eric and Dave.

**Review by RickJWagner — APPROVED** [review]
No objections, nothing to add to the conversation.  Acknowledging validity of the question about log volumes.
Adding conditional approval, pending satisfactory responses to the questions raised by colleagues.

**Pull request merged**